docs/reference/docs/bulk.asciidoc

```diff
@@ -154,13 +157,23 @@ The result of this bulk operation is:
                 "failed": 0
             },
             "status": 200,
-            "_seq_no" : 3
+            "_seq_no" : 3,
+            "_primary_term" : 4
          }
       }
    ]
 }
 --------------------------------------------------
-// TESTRESPONSE[s/"took": 30/"took": $body.took/ s/"index_uuid": .../"index_uuid": $body.items.3.update.error.index_uuid/ s/"_seq_no" : 0/"_seq_no" : $body.items.0.index._seq_no/ s/"_seq_no" : 1/"_seq_no" : $body.items.1.delete._seq_no/ s/"_seq_no" : 2/"_seq_no" : $body.items.2.create._seq_no/ s/"_seq_no" : 3/"_seq_no" : $body.items.3.update._seq_no/]
+// TESTRESPONSE[s/"took": 30/"took": $body.took/]
```

Comment:
Yes, I didn't know it was until @nik9000 made the suggestion and so I went and read the source to understand the syntax behind it.